docs/main.js

```diff
@@ -12,7 +12,7 @@ $(".language-jsx").each(function() {
     var prefill = {
         "title":"RJSF example",
         "tags":[],
-        "scripts":["https://cdnjs.cloudflare.com/ajax/libs/react/16.13.1/umd/react.production.min.js","https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.13.1/umd/react-dom.production.min.js","https://unpkg.com/@rjsf/core@2.5.1/dist/react-jsonschema-form.js"],
+        "scripts":["https://cdnjs.cloudflare.com/ajax/libs/react/17.0.2/umd/react.production.min.js","https://cdnjs.cloudflare.com/ajax/libs/react-dom/17.0.2/umd/react-dom.production.min.js","https://unpkg.com/@rjsf/core@3.0.1/dist/react-jsonschema-form.js"],
```

Comment:
Let's always get the latest version

```suggestion
        "scripts":["https://cdnjs.cloudflare.com/ajax/libs/react/17.0.2/umd/react.production.min.js","https://cdnjs.cloudflare.com/ajax/libs/react-dom/17.0.2/umd/react-dom.production.min.js","https://unpkg.com/@rjsf/core/dist/react-jsonschema-form.js"],
```